pulp_rpm/app/models/repository.py

```diff
@@ -102,6 +108,74 @@ class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
 
+class UlnRemote(Remote):
+    """
+    Remote for "uln" content.
+    """
+
+    TYPE = "uln"
+
+    uln_user = models.CharField(max_length=512, null=True)
+
+    uln_pwd = models.CharField(max_length=512, null=True)
+
+    uln_server_base_url = models.CharField(max_length=512, null=True)
+
+    @property
+    def download_factory(self):
+        """
+        Return the DownloaderFactory which can be used to generate asyncio capable downloaders.
+
+        Returns:
+            DownloadFactory: The instantiated DownloaderFactory to be used by
+                get_downloader()
+
+        """
+        try:
+            return self._download_factory
+        except AttributeError:
+            self._download_factory = DownloaderFactory(
+                self,
+                downloader_overrides={
+                    "http": RpmDownloader,
+                    "https": RpmDownloader,
+                    "file": RpmFileDownloader,
```

Comment:
Do we need to list more than just `uln` here?
@bmbouter , any thoughts?

Reply:
I think just listing `uln` as long as the `RpmDownloader` is configured as necessary by the `DownloaderFactory` subclass in RPM.

Reply:
I got rid of everything but `"uln"` as a test. (both versions currently run into the same error).